content/articles/learn-how-to-set-up-linux-quagga-routing-suite/index.md

```diff
@@ -0,0 +1,255 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /learn-how-to-set-up-linux-quagga-routing-suite/
+title: Learn how to set up Linux's Quagga Routing Suite
+description: In this tutorial, we will exhibit how to introduce the Quagga steering suite on Ubuntu working framework through an order line interface.
+author: bernard-mburu
+date: 2021-11-03T00:00:00-12:30
+topics: [Linux]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/learn-how-to-set-up-linux-quagga-routing-suite/hero.png
+    alt: Learn how to set up Linux's Quagga Routing Suite Hero Image
+---
+
+Quagga is a popular open source project for providing routing services on the Linux platform. It is made up of several components for various dynamic protocols.
+<!--more-->
+Such as, Open Shortest Path First (OSPF), Routing Information Protocol (RIP), Border Gateway Protocol (BGP), Intermediate System to Intermediate System (IS-IS), and Multiprotocol Label Switching (MPLS). It partially uses the same virtual terminal or CLI (vty/vtysh) for protocol configuration as CISCO/JUNIPER. We will install Quagga routing suite on Ubuntu in this article.
+### Objectives
+> By the end of this article you should be able to :
+- Know what is Quagga Linux and its brief history
+- Understand the working of Quagga Linux
+- Know why you should use Quagga linux
+- Supported Platforms by Quagga linux
+- Install Quagga in a Linux Environment
+- Configure the following:
+      1. Zebra daemon
+      2. Ospfd daemon
+      3. Ripd daemon
+### Table of contents
+- [Objectives](#objectives)
+- [Table of contents](#table-of-contents)
+- [A quick history of Quagga and an explanation of what it is](#a-quick-history-of-quagga-and-an-explanation-of-what-it-is)
+- [How does Quagga work?](#how-does-quagga-work)
+- [Who uses Quagga today, and what other open-source alternatives are available?](#who-uses-quagga-today-and-what-other-open-source-alternatives-are-available)
+- [Supported Platforms](#supported-platforms)
+- [Installation](#installation)
+- [Configuration](#configuration)
+  - [Zebra daemon](#zebra-daemon)
+- [Ospfd daemon](#ospfd-daemon)
+  - [Ripd daemon](#ripd-daemon)
+- [Conclusion](#conclusion)
+  - [Further reading](#further-reading)
+### A quick history of Quagga and an explanation of what it is
+Quagga is a GPLv2-licensed open-source routing stack. It's a program that implements IP routing protocols like `RIP`, `RIPng`, `OSPF`, and `ISIS`. I'd want to create a clear separation between a routing stack and a complete router solution. Traffic forwarding and a routing stack are required for a complete router. Only the routing protocols are implemented by Quagga. It may operate on Linux and forward traffic using the regular Linux kernel, or it can connect to a distributed forwarding platform using OpenFlow or another open or proprietary interface. It might also be used just for routing protocols to accept and announce routes using off-the-shelf routers.

```

Comment:
Thanks for adding this introduction but the flow and wording could be improved. Try to link the sentences and ideas together more rather than using lots of seperate sentences.

Reply:
@louisefindlay23 , suggested changes are in place. Thank you.

Reply:
@louisefindlay23, can we have the go ahead to review this tutorial if the changes made have covered the content adequately.